salt/addons/init.sls

```diff
@@ -14,40 +10,44 @@ include:
     - source:      salt://addons/addons/namespace.yaml.jinja
     - template:    jinja
 
-{% if salt['pillar.get']('addons:dns', 'false').lower() == 'true' %}
-/etc/kubernetes/addons/kubedns-sa.yaml:
-  file.managed:
-    - source:     salt://addons/addons/kubedns-sa.yaml.jinja
-    - template:   jinja
-
-/etc/kubernetes/addons/kubedns-cm.yaml:
-  file.managed:
-    - source:      salt://addons/addons/kubedns-cm.yaml.jinja
-    - template:    jinja
+apply-namespace:
+  cmd.run:
+    - name: |
+        kubectl apply -f /etc/kubernetes/addons/namespace.yaml
```

Comment:
The Salt master isn't configured with an appropriate kubeconfig file at this time. If we want to go that route, we need to open a separate story / Trello card for that, as there are several places that would benefit from that.